apps/universal/src/app/kitchen-sink/kitchen-sink.html

```diff
@@ -332,6 +332,8 @@ <h1>Packets</h1>
   <dt-step label="Step 3"></dt-step>
 </dt-stepper>
 
+<dt-slider></dt-slider>
```

Comment:
please specify all possible inputs so that universal has to run through them.

Reply:
Will do, thanks.